test/src/api/routes/test-publisher.js

```diff
@@ -153,3 +163,139 @@ describe('GET /Publisher', () => {
 	 });
 });
 
+/* eslint-disable no-await-in-loop */
+describe('Browse Publishers', () => {
+	let work;
+	before(async () => {
+		await truncateEntities();
+		// create 10 publishers 5 each of type 1 and type 2
+		const publisherBBIDs = [];
+		for (let areaId = 1; areaId <= 5; areaId++) {
```

Comment:
Why 5 of each where 2 of each would do?

Reply:
Umm I don't know why I did this. I'll change it to 2. :D